package.json

```diff
@@ -181,5 +179,14 @@
     "node": ">= 10.17.0",
     "npm": ">= 6.11.0",
     "postgres": ">= 9.4"
+  },
+  "vuePlugins": {
+    "service": [
+      "./build/plugins/vue-cli/brand",
+      "./build/plugins/vue-cli/clean-out-dir",
+      "./build/plugins/vue-cli/copy",
+      "./build/plugins/vue-cli/html-version-spec",
+      "./build/plugins/vue-cli/sass-resources"
```

Comment:
Can we swap this with https://github.com/ExtensionEngine/vue-cli-plugin-sass-resources 🤔

Reply:
Done